main.nf

```diff
@@ -189,16 +192,16 @@ Channel
     .into{ macs_para; saturation_para }
 
 /*
- * Reference to use for MACS and ngs.plot.r
+ * Reference to use for MACS, ngs.plot.r and annotation
  */
 def REF_macs = false
 def REF_ngsplot = false
-if (params.genome == 'GRCh37'){ REF_macs = 'hs'; REF_ngsplot = 'hg19' }
-else if (params.genome == 'GRCm38'){ REF_macs = 'mm'; REF_ngsplot = 'mm10' }
+if (params.genome == 'GRCh37'){ REF_macs = 'hs'; REF_ngsplot = 'hg19'; gtf = file(params.gtf) }
```

Comment:
I think we can define `gtf = file(params.gtf)` whatever the genome, right? That's one of the benefits of using a flat GTF reference, we can support any genome..